fastlane/Fastfile

```diff
@@ -200,3 +236,17 @@ def attach_changelog_to_main(options)
     }
   end
 end
+
+def current_version_number
+  version = nil
+
+  Dir.chdir(get_root_folder) do
+    version = (sh "awk -F\\\" \'/\"version\":\/ {print $4}\' package.json").strip
```

Comment:
Not needed for this PR but should we switch to the same versioning system we use in iOS of having a `.version` file containing just the version number? Just something to think about.

Reply:
I thought about it but it feels weird for a JS project because the `package.json` is the source of truth. In our case we update everything automatically though so it might not be the worst thing.

Reply:
I just filed https://revenuecats.atlassian.net/browse/CF-849 /cc @aboedo